src/main/java/com/netgrif/application/engine/workflow/web/CaseImportExportController.java

```diff
@@ -0,0 +1,107 @@
+package com.netgrif.application.engine.workflow.web;
+
+import com.netgrif.application.engine.configuration.properties.CaseImportExportProperties;
+import com.netgrif.application.engine.files.throwable.StorageException;
+import com.netgrif.application.engine.workflow.domain.Case;
+import com.netgrif.application.engine.workflow.exceptions.ImportXmlFileMissingException;
+import com.netgrif.application.engine.workflow.service.interfaces.ICaseImportExportService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.security.SecurityRequirement;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.core.io.InputStreamResource;
+import org.springframework.core.io.Resource;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@Slf4j
+@RestController()
+@RequestMapping("/api/case")
+@AllArgsConstructor
+public class CaseImportExportController {
```

Comment:
For consistency reasons, shouldn't there be two controllers? One is for export endpoints, and the other is for import endpoints.